spring-kafka/src/main/java/org/springframework/kafka/streams/KafkaStreamsInteractiveQueryService.java

```diff
@@ -78,29 +81,14 @@ public <T> T retrieveQueryableStore(String storeName, QueryableStoreType<T> stor
 		}
 		Assert.notNull(this.kafkaStreams, "KafkaStreams cannot be null");
 		StoreQueryParameters<T> storeQueryParams = StoreQueryParameters.fromNameAndType(storeName, storeType);
-		AtomicReference<StoreQueryParameters<T>> storeQueryParametersReference = new AtomicReference<>(storeQueryParams);
 
-		return getRetryTemplate().execute(context -> {
+		return this.retryTemplate.execute(context -> {
 			try {
-				return this.kafkaStreams.store(storeQueryParametersReference.get());
+				return this.kafkaStreams.store(storeQueryParams);
 			}
 			catch (Exception e) {
 				throw new IllegalStateException("Error retrieving state store: " + storeName, e);
 			}
 		});
 	}
```

Comment:
Still now new line after the class member 😉 .
I wonder if this can help somehow to be imported into your IDEA: https://github.com/spring-projects/spring-integration/blob/main/src/idea/spring-framework.xml

Reply:
Yes blank line.